database/postgres.go

```diff
@@ -18,6 +19,7 @@ func ConnectToDatabase(dbURL string) error {
 		SQL_DB, _ = db.DB()
 		DB_MIGRATOR = db.Migrator()
 
+		fmt.Println("DB_MIGRATOR", DB_MIGRATOR)
```

Comment:
Same, if we want to print something we should use a logger. fmt.Print is used for development purpose only